### PR TITLE
Updating documents

### PR DESCRIPTION
**Pull request by Th-429B**
Below is the list of updates:
1) Added contact details in About us.
2) Updated roles and responsibilities
3) Updated Use case scenarios (to include when fields exceed set limit)
4) Updated glossary
5) Update PPP (need to change the file name later)
6) Added manual testing for sort and saving data
7) Added sequence diagram
8) Updated DG for sort

Non-documentation updates: (I accidentally committed on this branch :( )
- Improve code quality (SLAP)
- Added logging

### CODE REVIEW

File: docs/diagrams/SortSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+activate SortCommand
+
+SortCommand --> SortCommandParser : sc
+deactivate SortCommand
+
+SortCommandParser --> FastParser : sc
+deactivate SortCommandParser
+'Hidden arrow to position the destroy marker below the end of the activation bar.
+SortCommandParser -[hidden]-> FastParser
+destroy SortCommandParser
```
Comment (squeakysquak) [inline docs/diagrams/SortSequenceDiagram.puml:53]: Is it possible to make the lifeline stop at the X?

**Review by squeakysquak — COMMENTED** [review]
One question about the diagram, other than that LGTM

File: docs/diagrams/SortSequenceDiagram.puml (hunk 1 above)
Comment (Th-429B) [reply docs/diagrams/SortSequenceDiagram.puml:53]: I don't think that's possible, referencing other groups DG

**Review by squeakysquak — APPROVED** [review]
Change no longer required, LGTM

**Pull request merged**